common/es/src/main/java/io/apiman/common/es/util/EsConstants.java

```diff
@@ -78,72 +80,324 @@
 
 
     public static final String[] MANAGER_INDEX_POSTFIXES = {
-            INDEX_MANAGER_POSTFIX_GATEWAY,
```

Comment:
Nit: indentation messing up the diff a bit, please match old indentation if possible!